II.pass-tests/10-paint-ford/paint-cars.js

```diff
@@ -0,0 +1,46 @@
+
+var cars = [
+    { make: 'Ford', model: 'Fiesta', colour: 'Red' },
+    { make: 'Land Rover', model: 'Defender', colour: 'Muddy' },
+    { make: 'Toyota', model: 'Prius', colour: 'Silver' },
+    { make: 'Honda', model: 'Civic', colour: 'Yellow' }
+];
+
+var unpaintedCars = [
+    { make: 'Ford', model: 'Fiesta', colour: 'Red' },
+    { make: 'Land Rover', model: 'Defender', colour: 'Muddy' },
+    { make: 'Toyota', model: 'Prius', colour: 'Silver' },
+    { make: 'Honda', model: 'Civic', colour: 'Yellow' }
+];
+
+var repaintedCars = [
+    { make: 'Ford', model: 'Fiesta', colour: 'Pink' },
+    { make: 'Land Rover', model: 'Defender', colour: 'Muddy' },
+    { make: 'Toyota', model: 'Prius', colour: 'Silver' },
+    { make: 'Honda', model: 'Civic', colour: 'Yellow' }
+];
+
+ function paintShop(array, color) {
```

Comment:
This function doesn't pass the test. It seems to be looking for cars that are already that colour, rather than "painting" any.